### ISSUE TITLE
Possible incompatibility with Flake8 >= 5.0.4

### ISSUE THREAD

**Issue by nkukard** [python-formate/flake8-dunder-all#44]
## Description
After upgrading flake I seem to be getting the below error...

```
There was a critical error during execution of Flake8:
plugin code for `flake8-dunder-all[DALL]` does not match ^[A-Z]{1,3}[0-9]{0,3}$
```

## Steps to Reproduce
   1. Install flake8 5.0.3
   2. Install flake8-dunder-all
   3. Try lint some python code

## Actual result:
```
There was a critical error during execution of Flake8:
plugin code for `flake8-dunder-all[DALL]` does not match ^[A-Z]{1,3}[0-9]{0,3}$
```

## Expected result:

No critical error.

## Reproduces how often:
Easily reproduced

## Version

  * Operating System: Arch
  * Python: 3.10.5
  * flake8: 5.0.4
  * flake8-dunder-all: 0.2.2

flake8 and flake8-dunder-all is installed in a tox environment using "deps = flake8 flake8-dunder-all".

**Comment by sshishov** [comment]
@domdfcoding is this repo still alive? can we merge the blocking MR?

**Comment by domdfcoding** [comment]
I have released version 0.3.0 which should fix this. Please reopen if you continue to have issues.
